protocol/signalfx/format/log/signalfx_log.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package sfx_log_format;
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "gen_go/;log_format";
+
+// model is based off https://github.com/open-telemetry/oteps/blob/master/text/0097-log-data-model.md#log-and-event-record-definition
+message LogRecord {
+    google.protobuf.Timestamp Timestamp = 1;
+    string TraceID = 2;
+    string SpanID = 3;
+    string TraceFlags = 4;
+    string SeverityTest = 5;
+    string Name = 6;
+    AttributeValue Body = 7;
+    map<string, AttributeValue> Resource = 8;
```

Comment:
You may want to extract Resource in a separate message and transmit it once per LogRecord batch.

Reply:
yeah that makes sense based on earlier discussion

Reply:
@tigrannajaryan i tried with example hopefully i am doing the correct thing

HecV1 Json data with resource type being unique per record:

`[ 
{
				TimeStamp:      1586960586000,
                                  Host : "macbook",
                                  Source: "home",
				Body:           "first", 
},
{
				TimeStamp:      1586960596000,
                                  Host : "build_server",
                                  Source: "office",
				Body:           "Second", 
},
]`

Protobuf representation of LogRequest will be as follows:

`
       LogRequest [
                 {
                        LogRecord : [ { Timestamp: 1586960586000, Body: "first" } ],
                       Resource: {"host":"macbook", "source":"home"},
                },
                 {
                        LogRecord : [ { Timestamp: 1586960596000, Body: "Second" } ],
                       Resource: {"host":"build_server", "source":"office"},
                },
        ]
`

Reply:
Yes, that's exactly correct. And if you have "fields" in HEC they will go into `Attributes` under the `LogRecord`.